lib/sync.js

```diff
@@ -95,6 +97,11 @@ function Sync(src, target, opts, callback) {
     };
   }
 
+  if (opts.live) {
+    this.push.on('complete', self.pull.cancel.bind(self.pull));
+    this.pull.on('complete', self.push.cancel.bind(self.push));
+  }
+
```

Comment:
That's weird; it seems like that ought to be handled already by [these lines](https://github.com/pouchdb/pouchdb/blob/ada62f35b7fa1badab9dcf8e9b50689dc13e4000/lib/sync.js#L178-L184).